src/main/java/busTickets/FindMaxViolation.java

```diff
@@ -0,0 +1,21 @@
+package busTickets;
+
+public class FindMaxViolation {
+    public static String findMaxViolation(int a, int b, int c) {
```

Comment:
This method has 2 responsibilities: 
1. find max from 3 values
2. mapping number-name of violation (also need to use meaningful names, it's not obvious that a stands for number of violation for start date)

so it should be 2 separate methods

Reply:
done